### DISCUSSION TITLE
UI: Dialog

### DISCUSSION THREAD

**Discussion by tmedwards** [tmedwards/sugarcube-3#37] [Ideas]
My current plan is to change the styling on the base dialog so that it takes up the full viewport, to give a more mobile/metro app feel to it rather than the current "I'm a window" feel.  Don't have a mockup to show, but you should get the idea I hope.

Thoughts?

**Comment by ChapelR** [comment]
I think SC should come with a snackbar/toast UI component too. I would be a fan of less window-y modals as well.

**Reply by tmedwards** [reply]
Toast notifications are on my like-to-have list.

**Comment by HiEv** [comment]
Perhaps do something like what I suggested in my '[Add a "popout" window](https://github.com/tmedwards/sugarcube-3/issues/25)' suggestion.  The same could be done for the UI menu.  Thus, "popout windows" would become the new version of "modal dialogs".

If you want, I could show you an example of how I've done it in some code I'm working on.
